lib/tasks/segment/segment.rake

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require_dependency 'csv'
+
+namespace :segment do
+  desc 'Save as deleted objects the users from the imported segment csv'
+  task :save_deleted_users, [:file_name] => [:environment] do |_task, args|
+    file_path =  Rails.root.join(args[:file_name])
```

Comment:
I think it didn't even need to be in the app directory anyway, but an absolute path instead.

Reply:
@guicassolato This isn't in the 'app' directory, this is anywhere inside the Rails root. It can be app, lib, config, etc.
However, if you mean that we should accept outside the rails root as well, ok :+1: It could be a feasible option :+1: